docs/pages/application-access/getting-started.mdx

```diff
@@ -23,38 +36,42 @@ Let's connect to Grafana using Teleport in three steps:
 
 - (!docs/pages/includes/tctl.mdx!)
 - A host where you will run the Teleport Application Service.
+- A DNS subdomain for each application.
```

Comment:
Most people will just use a wildcard DNS here - something like `*.teleport.example.com`.

Reply:
Yep, there's actually an include box that mentions this. I'm not a big fan of making these common "include" snippets in these assertive little boxes, but I didn't want to rock the boat **too** much in my first PR 😄 .
![image](https://github.com/gravitational/teleport/assets/136391445/ff670e08-5d9b-4db4-a6fa-6c9900ca8dc7)